packages/cactus-plugin-keychain-azure-kv/src/main/typescript/generated/openapi/typescript-axios/git_push.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
@jagpreetsinghsasan Please ignore this file in the openapi generator ignore file (and also remove the file itself from version control)